<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commonview</class>
 <widget class="QWidget" name="commonview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>872</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>92</red>
        <green>153</green>
        <blue>176</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>92</red>
        <green>153</green>
        <blue>176</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>92</red>
        <green>153</green>
        <blue>176</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>92</red>
        <green>153</green>
        <blue>176</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Радиолокационная система посадки</string>
  </property>
  <widget class="RightTriangleLocator" name="RenderRightTrinagleLocator" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="EquiangularTriangleLocator" name="RenderEquingularTriangleLocator" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="MainLocator" name="RenderMainLocator" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>430</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeRightTriangleFocus">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeRightTriangleFocus">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФОКУС</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeRightTriangleBrightness">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>30</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeRightTriangleBrightness">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ЯРКОСТЬ</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeMainLocatorBrightness">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>710</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeMainLocatorFocus">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>710</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorFocus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФОКУС</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorBrightness">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>690</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ЯРКОСТЬ</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeEquingularLocatorMagnificationSDC">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>780</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeEquingularLocatorMagnificationPASS">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>780</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeEquingularLocatorMagnificationAKT">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>780</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorMagnification">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>760</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>УСИЛЕНИЕ ПРИЁМНИКА</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorMagnificationSDC">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>820</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>СДЦ</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorMagnificationPASS">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>820</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ПАСС</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorMagnificationAKT">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>820</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>АКТ</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeEquingularLocatorScale">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>790</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/switch_base.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorScale">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>760</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>МАСШТАБ</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorScale60">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>775</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>60км</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorScale20">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>798</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>20км</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorScale30">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>820</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>30км</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeEquingularLocatorMode">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>800</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/knob);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorModePASS">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>780</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ПАСС</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorModeSMESH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>780</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>СМЕШ</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorModeAKT">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>780</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>АКТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeEquingularLocatorMode">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>830</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫБОР ВИДЕО</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorScale">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>800</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>МАСШТАБ</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorScale90">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>750</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>90км</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorScale150">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>750</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>150км</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorScale45">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>750</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>45км</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeMainLocatorScale">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>770</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/knob);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorMagnificationPASS">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>830</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ПАССИВ.</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorMagnificationAKT">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>830</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>АКТИВ.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeMainLocatorMagnificationPASS">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>790</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelChangeMainLocatorMagnification">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>770</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>УСИЛЕНИЕ ПР-КА</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ChangeMainLocatorMagnificationAKT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>790</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>289</width>
     <height>87</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="LabelChangeRightTriangleBrightnessMarks">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ЯРКОСТЬ МЕТОК&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QToolButton" name="ChangeRightTriangleBrightnessMarksRange">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>OpenHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="LabelChangeRightTriangleBrightnessMarksRange">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ДАЛЬНОСТИ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QToolButton" name="ChangeRightTriangleBrightnessMarksAzimuth">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>OpenHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/buttons/reo_knob.png);
background-repeat: no-repeat;
background-position: center;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="LabelChangeRightTriangleBrightnessMarksAzimuth">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;УГЛОВОЙ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="ChangeMainLocatorFocusSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeMainLocatorBrightnessSlider">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>720</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeMainLocatorMagnificationAKTSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>850</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeMainLocatorMagnificationPASSSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>850</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeEquingularLocatorMagnificationSDCSlider">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>850</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeEquingularLocatorMagnificationPASSSlider">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>850</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ChangeEquingularLocatorMagnificationAKTSlider">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>850</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainLocator</class>
   <extends>QWidget</extends>
   <header>mainlocator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EquiangularTriangleLocator</class>
   <extends>QWidget</extends>
   <header>equiangulartrianglelocator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RightTriangleLocator</class>
   <extends>QWidget</extends>
   <header>righttrianglelocator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
